etc/sdwdate.d/30_default.conf

```diff
@@ -30,15 +30,15 @@ MAX_FAILURE_RATIO=0.34
 ## pool one.
 ##    SecureDrop List
 SDWDATE_POOL_ONE=(
-         "secrdrop5wyphb5x.onion#https://securedrop.org	https://web.archive.org/web/20170403043247/https://securedrop.org"
-         "gmg7jl25ony5g7ws.onion#Gizmodo Media Group	https://specialprojectsdesk.com/securedrop/	https://web.archive.org/web/20170215221547/https://specialprojectsdesk.com/securedrop/"
```

Comment:
Cleranet link changed to https://specialprojectsdesk.com/secure-drop. Still online.